src/components/MedicineBasicSearch/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import tablet from '../../../static/images/pill_24.png';
+import capsule from '../../../static/images/capsule_24.png';
+import syrup from '../../../static/images/syrup_24.png';
+import classes from './style.module.css';
+
+const callImage = dosage_form => {
```

Comment:
A more functional programming way would be to do the following:

```
const calcIcon = (dosageForm) => {
  switch (dosageForm) {
    case 'suspension': return syrup;
    case 'tablet': return tablet;
    case 'effervescent tablet': return tablet;
    case 'capsule': return capsule;
    default: return tablet;
  }
}

const medIcon = calcIcon(dosage_form);
\
```

Reply:
Got it.